extensions/cornerstone-dicom-sr/src/tools/DICOMSRDisplayTool.ts

```diff
@@ -194,28 +197,15 @@ export default class DICOMSRDisplayTool extends AnnotationTool {
     }
   };
 
-  // repositionTextBox(toolData, eventData) {
-  //   const toolBoundingBoxes = [];
-
-  //   for (let i = 0; i < toolData.length; i++) {
-  //     const toolDataI = toolData[i];
-
-  //     const { textBox } = toolDataI.handles;
-  //     const { anchorPoints } = textBox;
-
-  //     const boundingBox = _getBoundingBoxFromAnchorPoints(anchorPoints);
-  //     // Get the textbox bounding locations.
-  //     // Get the tool extents.
-  //   }
-  // }
-
   renderPolyLine(
     svgDrawingHelper,
     viewport,
     renderableData,
     annotationUID,
     options
   ) {
+    // Todo: this needs to use the drawPolyLine from cs3D since it is implemented
+    // now, before it was implemented with a loop over drawLine which is hacky
```

Comment:
Soooo, can we just do it? The method is merged.

This method as is now is not only hacky, it doesn't scale for SVG (even though it did for canvas), you are creating thousands of dom elements here.